src/io/node.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+import process from 'node:process'
+import chalk from 'chalk'
+import nodeReleases from 'node-releases/data/release-schedule/release-schedule.json'
+
+function getLatestNodeVersion() {
+  const versions = Object.keys(nodeReleases)
+  const latestVersion = versions[versions.length - 1]
+  const versionWithoutV = latestVersion.slice(1)
+  const nodeVersion = versionWithoutV.split('.')[0]
+  return nodeVersion
+}
+
+function getNodeVersion() {
+  const nodeVersionWithoutV = process.version.slice(1)
+  const nodeVersion = nodeVersionWithoutV.split('.')[0]
+  return nodeVersion
+}
+
+function checkNode() {
+  const nodeVersion = getNodeVersion()
+  const nodeVersionData = nodeReleases[`v${nodeVersion}` as keyof typeof nodeReleases]
```

Comment:
You can get the node version via `semver.major`.

Reply:
Done. Thanks for pointing onto the semver methods! :slightly_smiling_face: